src/main/resources/META-INF/resources/settings/settings.js

```diff
@@ -41,6 +51,14 @@ function load() {
     }
 }
 
+function changeUserName() {
+    let user = window.prompt("Wie heißt du?","dein Name");
```

Comment:
Use `const` instead of `let` because you are not changing user.
Again why is it in German?
If the user cancels the promt `null` is returned which is why I recommend adding an check for this case:
```javascript
if (!user) return:
```